lib/active_record_shards/model.rb

```diff
@@ -64,6 +60,14 @@ def self.extended(base)
 
     private
 
-    attr_accessor :sharded
+    attr_writer :sharded
+
+    def sharded?
+      if defined?(@sharded)
```

Comment:
`@sharded` would only be defined if you called `.not_sharded` which would set it to `false`.  Now, it may also be defined by line 69.